### PR TITLE
LPS-121802

### PR DESCRIPTION
**Pull request by brianikim**
https://issues.liferay.com/browse/LPS-121802

### CODE REVIEW

File: modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/css/_ratings_animated.scss [hunk 1]
```diff
@@ -7,17 +7,8 @@
...
-		> .c-inner.c-inner.c-inner.c-inner {
-			&,
-			span {
-				font-size: 1rem;
-				line-height: 1;
-			}
-		}
-
 		.on {
+			display: flex;
```
Comment (brianikim) [inline modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/css/_ratings_animated.scss:11]: I do not think these implementations in _ratings_animated.scss are being utilized. I still see the buttons utilize a form of .c-inner from a different .scss.

File: modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/css/main.scss [hunk 2]
```diff
@@ -2,11 +2,17 @@
 
 .ratings {
 	.btn {
+		align-items: center;
+		display: inline-flex;
```
Comment (brianikim) [inline modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/css/main.scss:6]: I have noticed that the thumbs-up and thumbs-down are utilizing different css. I've seen this implementation be applied to only one of them.

File: modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/css/main.scss [hunk 3]
```diff
@@ -162,20 +162,59 @@
...
+				right: 0;
+				top: 0.25rem;
+				width: 100%;
+
+				> div {
+					float: left;
+				}
+			}
+		}
+
```
Comment (brianikim) [inline modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/css/main.scss:205]: I can confirm changing the position to static addresses the issue, but I am not sure col-md-4 (I also tried col-md-3) is the correct scope.

**Comment by brianikim** [comment]
Hello @pat270 ,

I wrote some observations that I saw with your recommended changes. Also, please let me know if my implementation of your suggestions were incorrect.

Sincerely,
Brian

File: modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/css/_ratings_animated.scss (hunk 1 above)
Comment (pat270) [reply modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/css/_ratings_animated.scss:11]: This css applies when you click the thumbs up or down button. It changes the icon and does the animations. I think this is also used for `type="like"` but I don't remember from earlier.

File: modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/css/main.scss (hunk 2 above)
Comment (pat270) [reply modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/css/main.scss:6]: This also applies to `type="like"`. There's no way to change it via UI in Portal (not that I could find). You need to manually change/create them it in the code and deploy.

File: modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/css/main.scss (hunk 3 above)
Comment (pat270) [reply modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/css/main.scss:205]: `col-md-4` is correct, should be 33.33333% width.

**Comment by pat270** [comment]
@brianikim This looks good to me. Can you forward this to the https://github.com/liferay-lima/liferay-portal/ repo?

**Pull request closed without merging**